src/service.rs

```diff
@@ -230,9 +232,12 @@ impl ServiceRequest {
     }
 
     #[doc(hidden)]
-    /// Set new app data container
+    /// Add app data container to request's resolution set.
     pub fn set_data_container(&mut self, extensions: Rc<Extensions>) {
```

Comment:
Since this is additive, should it now be called `add_data_container`?

Reply:
+1 for renaming.